dotnet/Simple.Sauce/IncorrectSafariVersionException.cs

```diff
@@ -4,5 +4,7 @@ namespace Sauce.Bindings
 {
     public class IncorrectSafariVersionException : Exception
     {
+        public IncorrectSafariVersionException() : base() { }
```

Comment:
Codacy found an issue: [Remove this redundant 'base()' call.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6319844)